src/app/window.cpp

```diff
@@ -74,6 +76,7 @@ QTabWidget *MainWindow::tabs_widget()
     launcher->setObjectName("Launcher");
     SettingsTab *settings = new SettingsTab(this);
     settings->setProperty("prevent_disable", true);
+	CameraTab* camera = new CameraTab( this );
```

Comment:
Apologies for the tabs-versus-spaces discrepancy. I will go through and fix all of those before merge.

Reply:
Fixed